src/components/resizeable-image.tsx

```
@@ -0,0 +1,178 @@
+} from "@tiptap/react";
+import {
+	type CSSProperties,
+	useCallback,
+	useEffect,
+	useLayoutEffect,
+	useRef,
+	useState,
+} from "react";
+import TipTapImage from "@tiptap/extension-image";
+
+const useEvent = <T extends (...args: any[]) => any>(handler: T): T => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)